scania-digital-design-system/sdds
# perf: bump versions for a lot of packages for node v16

**Describe pull-request**  
_Describe what the pull-request is about_

Preparation for node update [Node version LTS list](https://nodejs.org/en/about/releases/ )
- Commit that contains a lot of updates for different node packages, based on node v16 update.
- This will make it easier to run node v17 or v18(when it releases). 
- Packages that aren't update will be removed at some point and won't cause a problem
- Old components removed


**How to test**  
_Add description how to test if possible_
1. Run following commands: npm run clean, npm i, npm build, npm start, npm run link
2. Make sure everything runs, localhost
3. Linking works 
4. All the demos with linking packages
5. Make sure nothing is broken, and compare it to master 


**Additional context**  
_Add any other context about the pull-request here._
- This will add some depecrated message that we need to handle, but this will make sure reliable for node 16 and easier to upgrade to the next versions of node
- SASS requires some changes in the SASS that are being used in the grid
